modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -316,6 +316,10 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
             $vardef['name'] = 'aow_temp_date';
         }
 
+        if(!$vardef) {
```

Comment:
Yes this works, but it doesn't resolve the main issue or handles different types i.e. dates etc
The issue is that AOR doesn't deal with more than third level relationships - see here.
https://github.com/salesagility/SuiteCRM/blob/master/modules/AOR_Reports/controller.php#L543